components/formats-bsd/src/loci/formats/in/TiffReader.java

```diff
@@ -430,11 +430,12 @@ private void parseCommentGeneric(String comment) {
           addGlobalMeta(key, value);
         }
         else if (!line.startsWith("[")) {
-          comment += line + "\n";
+          buf.append(line);
+          buf.append("\n");
```

Comment:
Yes, that would save a few dereferences and internal method calls.  Using a typical size of OME-XML for us (about 700k lines), after JIT compilation, the string append for newlines takes about 3.5 milliseconds.  The char append for newlines takes about 2.5 milliseconds.  (All time measurements done on an Intel Xeon E5-1620 running at 3.5 GHz.)  This difference is negligible vs. the 10s of minutes in buffer allocation and deallocation and hundreds of thousands of multi-megabyte memory copies that I was fixing.

When performance is an issue, fix the big things.  The small things aren't worth the effort. ;-)